src/modules/app/Navigation.js

```diff
@@ -1,19 +1,17 @@
-// todo add logic for correct hrefs here
 class Navigation {
```

Comment:
I don't like this class very much, because it's mostly duplicate code in each function and is more some kind of configuration. Maybe we can make it come kind of json-compatible configuration (Or exported js-object). The configuration would be in each route then.

Possible problems: If you change something here the code in isCategoriesSelected() will fail.
Possible solution for this would be to test isCategoriesSelected() by using the paths form Navigation.js

Reply:
Hmm. yeah I don't like the class either... Was thinking about how to avoid it :D I think we need some better abstraction for all of our routes, so we can parse links more easily without such a class

Reply:
yeah, should we create a separate refactor task for this?

Reply:
we could do this in WEBAPP-90